server/src/Infrastructure/Persistence/Interceptors/AuditableEntityInterceptor.cs

```diff
@@ -1,24 +1,32 @@
-﻿using StudyZen.Domain.Interfaces;
+﻿using System.Security.Claims;
+using Microsoft.AspNetCore.Http;
+using StudyZen.Domain.Interfaces;
 using StudyZen.Domain.ValueObjects;
 
 namespace StudyZen.Infrastructure.Persistence;
 
 public static class AuditableEntityInterceptor
 {
-    public static void SetCreateStamp(object instance)
+    public static void SetCreateStamp(object instance, IHttpContextAccessor httpContextAccessor)
```

Comment:
This will break and conflict with the other PR so not sure whose problem it will.be 😃

Reply:
I believe there is a saying "The early bird gets the worm"